woocommerce/woocommerce
# feat(flexslider): no opacity anymore

### All Submissions:


### Changes proposed in this Pull Request:

Remove the opacity to the gallery that is reported as CLS from Lighthouse and is an old way to hide the gallery lazyload. Now WP and browser have a native way for images and a lot of things are happened in the last 6 years. Also after this https://github.com/woocommerce/woocommerce/pull/31407 some information are now in the page to improve the layout and size of the gallery.
The only doubts is that some CSS was inline in the templates folder, so as is removed now for all the themes around that copied still have this CSS. With those changes the opacity is not removed, so it is required a workaround to flip that value as true if opacity is set. Before to do that I am looking for feedbacks.

Closes #30893.

This changes is by me and @erikyo as we talked about how to do it in the best way.

### How to test the changes in this Pull Request:

1. Go to single product with Chrome
2. Check the CLS

### FOR PR REVIEWER ONLY:
